app/views/recipe_foods/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Add new Recipe</h1>
+
+ <%= form_with model: @recipe_new, url: recipe_recipe_foods_path do |f| %>
+  <div class="form_control">
+      <%= f.label :quantity %>
+      <%= f.number_field :quantity %>
+    </div>
+
+    <div class="form_control">
+    <%= f.select(:food_id, @foods.map{|u| [u.name, u.id]}) %>
+    </div>
+
+    <div class="form_control">
+    <%= f.hidden_field :recipe_id, value: @recipe_new.recipe_id %>
+    </div>
+    <%= f.submit "Submit", class:"btn btn-success" %>
+<% end%>
```

Comment:
- [ ] Just like the previous reviewer observed and advised, only authorized users (owner of the recipe) should be able to add a new food to the recipe as stated in the project requirements below 👇 
![image](https://user-images.githubusercontent.com/16392046/219865204-0ab52f4a-52eb-4045-83f6-c252516551aa.png)